packages/dev/hardhat.config.ts

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
I don't like how many commented-out lines are here, can we remove these comments?